src/test/java/teammates/test/cases/browsertests/InstructorEditStudentFeedbackPageUiTest.java

```diff
@@ -47,6 +53,23 @@ private void testModerationHint() throws Exception {
         assertEquals("[More]", submitPage.getModerationHintButtonText());
     }
 
+    private void testAddCommentsToQuestionsWithoutResponse() throws IOException {
+        ______TS("add comments on questions without responses: no effect");
+
+        logout();
+        submitPage = loginToInstructorEditStudentFeedbackPage(
+                "IESFPTCourseinstr", "student1InIESFPTCourse@gmail.tmt", "session1InIESFPTCourse");
+        submitPage.waitForPageToLoad();
+
+        submitPage.addFeedbackResponseComment("-3-0", "Comment without response");
+
+        submitPage.submitWithoutConfirmationEmail();
+        submitPage.verifyAndCloseSuccessfulSubmissionModal("2, 3.");
+        submitPage.waitForTextsForAllStatusMessagesToUserEquals(Const.StatusMessages.FEEDBACK_RESPONSES_SAVED,
+                Const.StatusMessages.FEEDBACK_UNANSWERED_QUESTIONS + "2, 3.");
+        submitPage.verifyHtmlMainContent("/instructorEditStudentFeedbackPageModifiedNoComments.html");
```

Comment:
Don't very main content. Use Page API to verify the content of the comment row.

Reply:
I haven't used it before. Can you tell me how to use it or point to source from where I can learn?

Reply:
You may find many examples in the existing code base.